pkg/asset/agent/manifests/agentclusterinstall_test.go

```diff
@@ -34,6 +37,13 @@ func TestAgentClusterInstall_Generate(t *testing.T) {
 			},
 			expectedError: "missing configuration or manifest file",
 		},
+		{
+			name: "install config has bad network type value",
+			dependencies: []asset.Asset{
+				installConfigWithBadNetworkType,
+			},
+			expectedError: "networkType has incorrect value. Expect value to be either 'OpenShiftSDN' or 'OVNKubernetes'",
```

Comment:
Does InstallConfig only allow those? I wonder what is set for other SDNs.

Reply:
There are a few more network types that are available: Kuryr, Raw, and SimpleMacvlan. 

https://github.com/openshift/api/blob/master/operator/v1/types_network.go#L598-613

Should we be supporting these other types?

Reply:
I think we should not prevent them from being set. Since people might also want to use things like calico and cilium

Reply:
i think you should allow other network types, such as Calico or Contrail, so not block on specific values

Reply:
Ok, I will update to allow other network types.